monai/apps/nnunet/nnunet_bundle.py

```diff
@@ -0,0 +1,448 @@
+    The returned nnUNet trainer can be used to initialize the SupervisedTrainer for training, including the network,
+    optimizer, loss function, DataLoader, etc.
+
+    Example::
+
+        from monai.apps import SupervisedTrainer
+        from monai.bundle.nnunet import get_nnunet_trainer
+
+        dataset_name_or_id = 'Task009_Spleen'
+        fold = 0
+        configuration = '3d_fullres'
+        nnunet_trainer = get_nnunet_trainer(dataset_name_or_id, configuration, fold)
+
+        trainer = SupervisedTrainer(
+            device=nnunet_trainer.device,
+            max_epochs=nnunet_trainer.num_epochs,
+            train_data_loader=nnunet_trainer.dataloader_train,
+            network=nnunet_trainer.network,
+            optimizer=nnunet_trainer.optimizer,
+            loss_function=nnunet_trainer.loss_function,
+            epoch_length=nnunet_trainer.num_iterations_per_epoch,
+        )
+
+    Parameters
+    ----------
+    dataset_name_or_id : Union[str, int]
+        The name or ID of the dataset to be used.
+    configuration : str
+        The configuration name for the training.
+    fold : Union[int, str]
+        The fold number or 'all' for cross-validation.
+    trainer_class_name : str, optional
+        The class name of the trainer to be used. Default is 'nnUNetTrainer'.
+    plans_identifier : str, optional
+        Identifier for the plans to be used. Default is 'nnUNetPlans'.
+    pretrained_weights : str, optional
+        Path to the pretrained weights file.
+    num_gpus : int, optional
+        Number of GPUs to be used. Default is 1.
+    use_compressed_data : bool, optional
+        Whether to use compressed data. Default is False.
+    export_validation_probabilities : bool, optional
+        Whether to export validation probabilities. Default is False.
+    continue_training : bool, optional
+        Whether to continue training from a checkpoint. Default is False.
+    only_run_validation : bool, optional
+        Whether to only run validation. Default is False.
+    disable_checkpointing : bool, optional
+        Whether to disable checkpointing. Default is False.
+    val_with_best : bool, optional
+        Whether to validate with the best model. Default is False.
+    device : str, optional
+        The device to be used for training. Default is 'cuda'.
+    pretrained_model : str, optional
+        Path to the pretrained model file.
+    Returns
+    -------
+    nnunet_trainer
+        The nnUNet trainer instance.
+    """
+    # From nnUNet/nnunetv2/run/run_training.py#run_training
+    if isinstance(fold, str):
+        if fold != "all":
+            try:
+                fold = int(fold)
+            except ValueError as e:
+                print(
+                    f'Unable to convert given value for fold to int: {fold}. fold must bei either "all" or an integer!'
+                )
+                raise e
+
+    if int(num_gpus) > 1:
```

Comment:
It seems we don't need support `num_gpus>1` in this `get_nnunet_trainer` function?